lib/scala/logging-config/src/main/java/org/enso/logging/config/systemlogger/ContextLoggingViaSlf4j.java

```diff
@@ -8,10 +8,12 @@
 import org.enso.logger.JulHandler;
 import org.enso.logging.config.LoggerSetup;
 import org.graalvm.polyglot.Context;
+import org.openide.util.lookup.ServiceProvider;
 
 /**
  * Provides implementation of the {@link ContextLoggingConfigurator} for the `engine-common` module.
  */
+@ServiceProvider(service = ContextLoggingConfigurator.class)
```

Comment:
@Akirathan, the d82036a22bbf5ca7555ccb28855bb8b81bd24b00 wouldn't be necessary, if we configured native image to use `--module-path` - I know you reported an issue about it, I just cannot find it now...